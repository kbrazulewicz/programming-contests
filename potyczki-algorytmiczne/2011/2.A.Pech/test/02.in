234 12345678 1234
