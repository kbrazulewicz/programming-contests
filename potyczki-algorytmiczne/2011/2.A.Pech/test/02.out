8449029.518
